### PR TITLE
Added section for camelCase & SnakeCase variables

### PR DESCRIPTION
**Pull request by keltonkowalchuk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,22 +40,40 @@
...
+	<cfargument name="MyObjectId" type="string" required="true">
+</cffunction>
+
+# good
+<cffunction name="init" access="package" output="false" returntype="CartObject">
+	<cfargument name="myObjectID" type="string" required="true">
+	...
+</cffunction>
+
+# bad - Id should be ID
```
Comment (lucasklaassen) [inline README.md]: Request wording change to be less redundant: # bad - `Id` should be uppercase

**Comment by keltonkowalchuk** [comment]
@lucasklaassen - Change made!

File: README.md [hunk 2]
```diff
@@ -40,22 +40,40 @@
 ```
 
 * <a name="camelCase"></a>
-  Use **camelCase** for variables, functions, and argument names. Use **SnakeCase** for object names.
+  Use **camelCase** for variables, functions, and argument names.
 <sup>[[link](#camelCase)]</sup>
 ```Coldfusion
-# bad - Object with camelCase and function/argument with SnakeCase
+# bad - Function/argument with SnakeCase
+<cffunction name="Init" access="package" output="false" returntype="CartObject">
```
Comment (lucasklaassen) [inline README.md]: Maybe we could keep the example focused on the core issue:

```Coldfusion
<cffunction name="Init">
    <cfargument name="MyObjectId">
</cffunction>
```

**Review by lucasklaassen — CHANGES_REQUESTED** [review]
Comment to make it more readable on first glance if being linked to.

**Comment by keltonkowalchuk** [comment]
@lucasklaassen - One more 🗡

**Review by lucasklaassen — APPROVED** [review]
LGTM

**Pull request merged**